airgun/widgets.py

```diff
@@ -1,32 +1,32 @@
 from cached_property import cached_property
 from wait_for import wait_for
-from widgetastic.exceptions import NoSuchElementException
-from widgetastic.exceptions import WidgetOperationFailed
-from widgetastic.widget import Checkbox
-from widgetastic.widget import ClickableMixin
-from widgetastic.widget import do_not_read_this_widget
-from widgetastic.widget import GenericLocatorWidget
-from widgetastic.widget import ParametrizedLocator
-from widgetastic.widget import Select
-from widgetastic.widget import Table
-from widgetastic.widget import Text
-from widgetastic.widget import TextInput
-from widgetastic.widget import View
-from widgetastic.widget import Widget
+from widgetastic.exceptions import NoSuchElementException, WidgetOperationFailed
+from widgetastic.widget import (
+    Checkbox,
+    ClickableMixin,
+    GenericLocatorWidget,
+    OUIAContextSelector,
+    ParametrizedLocator,
+    Select,
+    Table,
+    Text,
+    TextInput,
+    View,
+    Widget,
+    do_not_read_this_widget,
+)
 from widgetastic.xpath import quote
-from widgetastic_patternfly import AggregateStatusCard
-from widgetastic_patternfly import Button
-from widgetastic_patternfly import FlashMessage
-from widgetastic_patternfly import FlashMessages
-from widgetastic_patternfly import Kebab
-from widgetastic_patternfly import VerticalNavigation
-from widgetastic_patternfly4.ouia import BaseSelect
-from widgetastic_patternfly4.ouia import Button as PF4Button
-from widgetastic_patternfly4.ouia import ContextSelector as OUIAContextSelector
-from widgetastic_patternfly4.ouia import Dropdown
-
-from airgun.exceptions import DisabledWidgetError
-from airgun.exceptions import ReadOnlyWidgetError
+from widgetastic_patternfly import (
+    AggregateStatusCard,
+    Button,
+    FlashMessage,
+    FlashMessages,
+    Kebab,
+    VerticalNavigation,
+)
+from widgetastic_patternfly4.ouia import BaseSelect, Button as PF4Button, Dropdown
```

Comment:
missing the `from widgetastic_patternfly4.ouia import ContextSelector as OUIAContextSelector` here